### DISCUSSION TITLE
`Get-WinEvent` should be abstracted with `journalctl` into `Get-SystemEvent`.

### DISCUSSION THREAD

**Discussion by roke-julian-lockhart** [PowerShell/PowerShell#24057] [Enhancements]
Considering that `Get-WinEvent` already exists on Windows, it should be about as simple as:

```PS1
#!/usr/bin/env -S pwsh

#```PS1
#Requires -PSEdition Core
#Requires -Version 7.4
#```

Set-StrictMode -Version 3.0

If ([OperatingSystem]::IsWindows() -Eq $True) {
    Get-WinEvent }

ElseIf ([OperatingSystem]::IsLinux() -Eq $True) {
    journalctl --output=json --no-pager | ConvertFrom-JSON }

**Comment by rhubarb-geek-nz** [comment]
I suggest if you want to attempt to unify the two views then this is done as a PSGallery module.

While superficially they may appear to be similar, the actual data returned is completely different

Your example on Windows gave me

```
Get-WinEvent: Log count (456) is exceeded Windows Event Log API limit (256). Adjust filter to return less log names.
```

So there would need to be some form of filtering that is common to both.

**Reply by roke-julian-lockhart** [reply]
Thanks, @rhubarb-geek-nz.
